runtime/entries/vue.js

```diff
@@ -17,7 +17,45 @@
  * under the License.
  */
 
-import setup from './setup'
-import * as Vue from 'weex-vue-framework'
+const instanceOptions = {}
```

Comment:
The `instanceOptions` variable will always exist even pages are destroyed, it may cause a potential memory leak, we should figure out another way to implement instance lifecycles.